packages/react/src/transactions/useRecentTransactions.ts

```diff
@@ -0,0 +1,19 @@
+import { uniqBy } from 'lodash';
+import { useSharedDependencies } from '../shared';
+import { TxStatus, useRecentTransactionsVar } from './adapters/TransactionQueuePresenter';
+
+export function useRecentTransactions() {
+  const { transactionQueue } = useSharedDependencies();
+
+  const transactions = useRecentTransactionsVar();
+
+  return {
+    clearCompleted() {
+      transactionQueue.clearRecent();
+    },
+    transactions: uniqBy(transactions, (t) => JSON.stringify(t.request)).map((transaction) => ({
+      ...transaction,
+      completed: transaction.status === TxStatus.SETTLED,
```

Comment:
Not sure if we want to expose the `TxStatus` enum since it seems a bit low level for someone consuming the react SDK.

Reply:
I see you point, one of the usages we were making of this data was to show the difference between: "pending", "failed" and "settled" (successfully).

`TxStatus` here is an enum designed for UI state reason, I personally think it's ok to surface at least `FAILED` and `SETTLED` options. Not 100% sure about `BROADCASTING` and `MINING` myself.

Soon we will add a feature (which you might have heard under the name DA - Data Availability) that might stir these concepts a little. Let's review this idea after that?